allennlp/common/tee_logger.py

```diff
@@ -10,7 +10,7 @@ class TeeLogger:
         sys.stdout = TeeLogger("stdout.log", sys.stdout)
```

Comment:
I'd say it's probably not worth our time trying to make mypy happy with the `TeeLogger`.